packages/node-client/lib/index.ts

```diff
@@ -174,15 +210,14 @@ export class Nango {
     }
 
     /**
-     * For OAuth 2: returns the access token directly as a string.
+     * For OAuth 2: returns the access token directly as a string
      * For OAuth 2: If you want to obtain a new refresh token from the provider before the current token has expired,
-     * you can set the forceRefresh argument to true."
-     * For OAuth 1: returns an object with 'oAuthToken' and 'oAuthTokenSecret' fields.
-     * @param providerConfigKey - This is the unique Config Key for the integration
-     * @param connectionId - This is the unique connection identifier used to identify this connection
-     * @param [forceRefresh] - When set, this is used to  obtain a new refresh token from the provider before the current token has expired,
-     * you can set the forceRefresh argument to true.
-     * */
+     * you can set the forceRefresh argument to true
+     * For OAuth 1: returns an object with 'oAuthToken' and 'oAuthTokenSecret' fields
+     * @param {string}providerConfigKey - The integration ID used to create the connection (aka Unique Key)
```

Comment:
```suggestion
     * @param {string}providerConfigKey - The integration ID used to create the connection (i.e Unique Key)
```

Reply:
@bodinsamuel , you are absolutely correct, since we also ship the d.ts Typescript which does show the actual types, there is no need of having to declare types again in jsdocs comments, this will make it easier to maintain. I have also updated my pull request. Thank you Samuel for that 👍